README.md

```diff
@@ -1,6 +1,18 @@
 # jupyterlab-pullrequests
 
-A JupyterLab extension for viewing and commenting on pull requests
+[![Build Status](https://travis-ci.org/jupyterlab/pull-requests.svg?branch=master)](https://travis-ci.org/jupyterlab/pull-requests)
+
+A JupyterLab extension for reviewing pull requests
+
+![](https://user-images.githubusercontent.com/9003282/62256798-a3ccb600-b3b7-11e9-921b-d72dcbb81477.gif)
+
+## Prerequisites
+
+- JupyterLab  
+
+## Usage
+
+- Open the pull request extension from the tab on the left panel
 
 ## Installation
```

Comment:
Add the installation instructions directly here so that someone interested in the extension doesn't need to do much guesswork and just copy-paste the installation commands.